csv_to_json.go

```diff
@@ -13,7 +13,7 @@ import (
 func csvToJSON(context *cli.Context) error {
 	var in *os.File
 
-	if infile := context.String(InputFileFlag.Names()[0]); len(infile) < 1 {
+	if infile := context.String("input-file"); len(infile) < 1 {
```

Comment:
This should use the `*Flag.Name` or `*Flag.Names()[0]` to "gracefully" handle the renaming of the flags rather than hardcoding the string in 2 locations.

Reply:
then, might be good to set a constfor each flag?

Reply:
A struct (unfortunately) can't be set as a const.